src/drivers/general/queries/ExtendedDeprecatedApis/driver_snippet.c

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// driver_snippet.c
+//
+
+#define SET_DISPATCH 1
```

Comment:
Where are you checking this?

Reply:
In the driver template code that this file is transposed into by Abenezer's testing script (https://github.com/microsoft/Windows-Driver-Developer-Supplemental-Tools/blob/development/src/drivers/PortFolder/WDMTestingTemplate/driver/fail_driver1.c).